src/main/java/EventTask.java

```diff
@@ -1,10 +1,14 @@
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
 public class EventTask extends Task {
     private final String taskTitle;
     private final String taskType = "E";
     private boolean markedStatus;
```

Comment:
Great adherence to the coding standards thus far, just something small that I noticed, take note that boolean variables should be revised to sound boolean. :)